lib/axiom/fs/stdio.js

```diff
@@ -47,3 +47,15 @@ export var Stdio = function(stdin, stdout, stderr, signal, ttyin, ttyout) {
 };
 
 export default Stdio;
+
+/**
+ * @return {void}
+ */
+Stdio.prototype.close = function() {
+  this.stdin.close();
+  this.stdout.close();
```

Comment:
For WritableStream, it should be a method named `end` (see https://nodejs.org/api/stream.html). I think we can leave the `end` method without any argument for now, unlike node.js streams.